src/java/org/apache/cassandra/gms/FailureDetector.java

```diff
@@ -93,16 +95,14 @@ public FailureDetector()
 
     private static long getInitialValue()
     {
-        String newvalue = System.getProperty("cassandra.fd_initial_value_ms");
-        if (newvalue == null)
-        {
+        Long newValue = FD_INITIAL_VALUE_MS.getLong();
+        if (newValue == null)
             return Gossiper.intervalInMillis * 2;
-        }
-        else
-        {
-            logger.info("Overriding FD INITIAL_VALUE to {}ms", newvalue);
-            return Integer.parseInt(newvalue);
-        }
+
+        if (newValue != Gossiper.intervalInMillis * 2)
```

Comment:
I do not get why we are doing here that `if`. The previous code is:

````
    private static long getInitialValue()
    {
        String newvalue = System.getProperty("cassandra.fd_initial_value_ms");
        if (newvalue == null)
        {
            return Gossiper.intervalInMillis * 2;
        }
        else
        {
            logger.info("Overriding FD INITIAL_VALUE to {}ms", newvalue);
            return Integer.parseInt(newvalue);
        }
    }
````

We log it every time `newvalue` is not null. Why do we set that condition for logging when it is not equal two times internvalInMillis?

Reply:
The default value is `Gossiper.intervalInMillis * 2`, so if the `FD_INITIAL_VALUE_MS` is set to the default, we don't want to log it, do we? 

This was the previous implementation:
```
    private static long getMaxLocalPause()
    {
        if (System.getProperty("cassandra.max_local_pause_in_ms") != null)
        {
            long pause = Long.parseLong(System.getProperty("cassandra.max_local_pause_in_ms"));
            logger.warn("Overriding max local pause time to {}ms", pause);
            return pause * 1000000L;
        }
        else
            return DEFAULT_MAX_PAUSE;
    }
```